test/workers/validator_check_active_worker_test.rb

```diff
@@ -2,11 +2,14 @@
 require 'sidekiq/testing'
 
 class ValidatorCheckActiveWorkerTest < ActiveSupport::TestCase
+  setup do
```

Comment:
All these tests should be moved to `test/services`. Here we want to only check if worker is properly added to queue etc,